slurp.py

```diff
@@ -372,23 +372,27 @@ def submit( rule, **kwargs ):
 
     jobd = rule.job.dict()
 
-    # f"{math.floor(v/n)*n}"
-    rundirs = []
-    for r in runlist:
-        d = f"{math.floor(r/100)*100}"
-        if d not in rundirs:
-            rundirs.append(d)
-
-    # Make sure that directories referenced in the filesystem are available
-    for k in [ 'outdir', 'logdir', 'condor']:
-        v = kwargs.get(k,None)
-        if v==None: continue
-        v = v.replace('file:/','')
-        v = v.replace('//','/')
-        v = v.split('$$')[0]
-        paths = [ v + '/' + d for d in rundirs ]
-        for p in paths: 
-            pathlib.Path( p ).mkdir( parents=True, exist_ok=True )        
+
```

Comment:
Specifies (in a fragile way) the name of the output directory as run_<run#min>_<run#max+1>.